packages/playwright-core/src/server/webkit/wkPage.ts

```diff
@@ -762,6 +766,10 @@ export class WKPage implements PageDelegate {
     await this._updateBootstrapScript();
   }
 
+  async removeInitScripts() {
```

Comment:
I struggle to see who calls this one 😄

Reply:
await this._delegate.removeInitScripts() in Page;